From 680e0718b9f0b9f3067b2c3947088c2878f55442 Mon Sep 17 00:00:00 2001
From: Liang Luo <luoliang@kylinos.cn>
Date: Fri, 31 Jul 2026 10:30:04 +0800
Subject: [PATCH 09846/15333] sched_ext: Fix stale @cgroup_id in sched_ext_ops
 kernel-doc

The kernel-doc comment for sched_ext_ops::sub_cgroup_id uses the old
@cgroup_id name, which no longer matches the struct member. This
produces two kernel-doc warnings:

  Warning: struct member sub_cgroup_id not described in sched_ext_ops
  Warning: Excess struct member cgroup_id description in sched_ext_ops

Update the @param name to match the actual member.

Signed-off-by: Liang Luo <luoliang@kylinos.cn>
Signed-off-by: Tejun Heo <tj@kernel.org>
---
 kernel/sched/ext/internal.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kernel/sched/ext/internal.h b/kernel/sched/ext/internal.h
index 7501ec28958b..741d8f6b5199 100644
--- a/kernel/sched/ext/internal.h
+++ b/kernel/sched/ext/internal.h
@@ -925,8 +925,8 @@ struct sched_ext_ops {
 	u32 cid_shard_size;
 
 	/**
-	 * @cgroup_id: When >1, attach the scheduler as a sub-scheduler on the
-	 * specified cgroup.
+	 * @sub_cgroup_id: When >1, attach the scheduler as a sub-scheduler
+	 * on the specified cgroup.
 	 */
 	u64 sub_cgroup_id;
 
-- 
2.55.0

